aspnet/web-forms/overview/getting-started/hands-on-labs/using-page-inspector-in-visual-studio-2012.md

```diff
@@ -212,36 +212,36 @@ In this task, you will use the Page inspector and fix some issues the Photo Gall
 
     Notice that the **#login** style is automatically located in **Site.css** after you click. Moreover, the code is now highlighted.
 
-    ![Selecting the CSS styles](using-page-inspector-in-visual-studio-2012/_static/image16.png "Selecting the CSS styles")
+    ![Screenshot of the Styles tab on the navigation bar the CSS styles for login is selected with the corresponding code highlighted.](using-page-inspector-in-visual-studio-2012/_static/image16.png "Selecting the CSS styles")
```

Comment:
Screenshot of the Styles tab, on the navigation bar the CSS styles for login is selected with the corresponding code highlighted.